### DISCUSSION TITLE
Unable to parse source.

### DISCUSSION THREAD

**Discussion by hkg11gold** [eclipse/windowbuilder#692] [Q&A]
![Screenshot (1766)](https://github.com/eclipse-windowbuilder/windowbuilder/assets/98084601/96467779-fd69-4407-b63a-7ae89a53cd10)

when I drop any option it says the same. I tried following the message, but when I deleted my import lines, the program returned an error. How should I fix it?

**Comment by ptziegler** [comment]
Can you provide the source code or at least a small reproducer? With just the screenshot, I can only speculate what the problem might be...
